ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/statetransition/epoch/AbstractEpochProcessor.java

```diff
@@ -41,29 +43,33 @@
 
 public abstract class AbstractEpochProcessor implements EpochProcessor {
   protected final SpecConfig specConfig;
+  protected final MiscHelpers miscHelpers;
   protected final ValidatorsUtil validatorsUtil;
   protected final BeaconStateUtil beaconStateUtil;
   protected final ValidatorStatusFactory validatorStatusFactory;
   protected final BeaconStateAccessors beaconStateAccessors;
 
   protected AbstractEpochProcessor(
       final SpecConfig specConfig,
+      final MiscHelpers miscHelpers,
       final ValidatorsUtil validatorsUtil,
       final BeaconStateUtil beaconStateUtil,
       final ValidatorStatusFactory validatorStatusFactory,
       final BeaconStateAccessors beaconStateAccessors) {
     this.specConfig = specConfig;
+    this.miscHelpers = miscHelpers;
     this.validatorsUtil = validatorsUtil;
     this.beaconStateUtil = beaconStateUtil;
     this.validatorStatusFactory = validatorStatusFactory;
     this.beaconStateAccessors = beaconStateAccessors;
   }
 
-  private static void applyDeltas(final MutableBeaconState state, final Deltas attestationDeltas) {
+  protected static void applyDeltas(
```

Comment:
Seems weird that this is static and I don't think it needs to be.

Reply:
👍 updated